cmd/metal-api/internal/service/machine-service.go

```diff
@@ -1690,6 +1751,8 @@ func (r machineResource) provisioningEventForMachine(machineID string, e v1.Mach
 	} else if event.Event == metal.ProvisioningEventPhonedHome && len(ec.Events) > 0 && ec.Events[0].Event == metal.ProvisioningEventPhonedHome {
 		zapup.MustRootLogger().Sugar().Debugw("swallowing repeated phone home event", "id", ec.ID)
 		ec.Liveliness = metal.MachineLivelinessAlive
+	} else if event.Event == metal.ProvisioningEventReinstallAborted {
```

Comment:
This way you swallow the event, which is "ok". But maybe it would be nice to track it in the state machine because then a user could see what went wrong in machine describe. And it would also be added to the machine issues, which could be beneficial for operations.